lib/Field/stories/FieldStory.js

```diff
@@ -36,64 +43,84 @@ storiesOf('Components', module).add('Field', () => {
+  const instructions = text(
+    'Instructions',
+    "Instruction text to help inform the user of what they're meant to write in this field. http://gathercontent.com"
+  );
+  const hasFormatting = boolean('Has formatting', true);
+  const disabled = boolean('Disabled', false);
+  const repeatPosition = number('Repeat position', 1);
+  const isLastRepeat = boolean('Is last repeat', false);
+  const fieldId = '123';
   return (
     <StoryItem
       title="Field"
       description="A field component provides supporting UI for gathering content. The child component should be responsible for handling the gathering experience but this component renders labels, actions and validations to support that experience."
     >
       <Field
-        fieldId="123"
-        actions={mockActions}
-        validations={mockValidations}
-        label={text('Field label', 'Field label text')}
-        instructions={text(
-          'Instructions',
-          "Instruction text to help inform the user of what they're meant to write in this field. http://gathercontent.com"
-        )}
-        hasFormatting={boolean('Has formatting', true)}
-        isActive={boolean('Is active', true)}
-        disabled={boolean('Disabled', false)}
-        canEdit={canEdit}
+        fieldId={fieldId}
+        hasFormatting={hasFormatting}
+        isActive={isActive}
+        disabled={disabled}
+        inStructureEditMode={inStructureEditMode}
         dir={dir}
       >
-        {!canEdit ? (
-          <div>
-            <h1>Heading content</h1>
-            <p>
-              The fossil record indicates that birds evolved from feathered
-              ancestors within the theropod group, which are traditionally
-              placed within the saurischian dinosaurs, though a 2017 paper[4]
-              has put them in a proposed clade Ornithoscelida, along with the
-              Ornithischia.
-            </p>
-            <table>
-              <tbody>
-                <tr>
-                  <td>
-                    <p>
-                      <strong>Heading 1</strong>
-                    </p>
-                  </td>
-                  <td>
-                    <p>
-                      <strong>Heading 2</strong>
-                    </p>
-                  </td>
-                </tr>
-                <tr>
-                  <td>
-                    <p>Cell 1</p>
-                  </td>
-                  <td>
-                    <p>Cell 2</p>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
-        ) : null}
+        <FieldLeft
+          className={isRepeatable && !inStructureEditMode ? '' : 'pt-6'}
+        >
+          {isRepeatable && !inStructureEditMode && (
+            <RepeatableSettings
+              isLastRepeat={isLastRepeat}
+              repeatPosition={repeatPosition}
+            />
+          )}
+          <FieldLabel label={label} />
```

Comment:
Should this be implemented as `Field.Label` as well as all the other components?

It doesn't make a massive difference other than clarity of structure.